eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/log/EventLog.scala

```diff
@@ -575,7 +575,7 @@ abstract class EventLog[A <: EventLogState](id: String) extends Actor with Event
         // 100% filtering coverage for certain replication network topologies.
         acc
       case (acc, e) =>
-        snr += 1L
+        snr = (snr + 1) max e.vectorTimestamp.value.getOrElse(id, 0)
```

Comment:
This breaks the algorithm for event log partitioning (used in `CassandraEventLog`). I propose to adjust the local sequence number from the log's version vector, if needed, after successful completion of recovery. This requires a new event log command for updating the local sequence number. The handler of this command should respect partition sizes (see also `EventLog.adjustSequenceNr`).